### PR TITLE
Resized gif in README.rst

### PR DESCRIPTION
**Pull request by bigboynaruto**
Closes: #43

### CODE REVIEW

File: demos/moderator/README.rst [hunk 1]
```diff
@@ -15,8 +15,7 @@ intensive tasks from blocking event loop.
 Install the app::
 
     $ cd moderator
-    $ pip -r requirements-dev.txt
-    $ pip install -e .
```
Comment (webknjaz) [inline demos/moderator/README.rst]: This change is unrelated

**Comment by bigboynaruto** [comment]
Done.

**Comment by webknjaz** [comment]
Thanks @bigboynaruto!
